libc-test/build.rs

```diff
@@ -289,6 +298,24 @@ fn main() {
             "FILE_ATTRIBUTE_INTEGRITY_STREAM" |
             "ERROR_NOTHING_TO_TERMINATE" if mingw => true,
 
+            // these are not in musl
+            "SOL_NETLINK" |
+            "SOL_UDP" |
+            "SOL_AX25" |
+            "SOL_ATALK" |
+            "SOL_NETROM" |
+            "SOL_IPX" |
+            "SOL_ROSE" if musl => true,
+
+            // these are not in glibc or musl yet (but they are in android)
+            "SOL_SCTP" |
+            "SOL_UDPLITE" |
+            "SOL_NETBEUI" |
+            "SOL_LLC" |
+            "SOL_DCCP" |
+            "SOL_NETLINK" |
+            "SOL_TIPC" if linux => true,
```

Comment:
Understood.  I'll see what I can come up with.  I would be fine with dropping all of these as well except `SOL_NETLINK` if it comes to that.  They are tested in Android, so there is some validation going on (but we do not validate Android on non-ARM).